easybuild/easyconfigs/t/tsne/tsne-0.1.8-intel-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,35 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'tsne'
+version = '0.1.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/danielfrg/tsne'
+description = "Python library containing T-SNE algorithms."
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'openmp': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+patches = ['%(name)s-%(version)s_fix_setup_py.patch']
```

Comment:
@Darkless012 Since the patch is now specific to `imkl`, the patch name should reflect that imho.

How about something like `tsne-0.1.8_fix-setup-py-imkl.patch`

Reply:
point taken

Reply:
@Darkless012 You marked this as resolved, but the patch file is not renamed yet?